src/view/configuration/configuration.jsx

```diff
@@ -474,6 +476,16 @@ const Configuration = () => {
                             </div>
                           ) : null}
 
+                            <div className="u-gapTop">
+                            <InfoTipLayout tip="Copy id from a previous version of experience cloud ID service.">
```

Comment:
"id" should probably be "ID". I'm guessing "experience cloud ID service" should be "Adobe Experience Cloud Identity Service" for branding/legal reasons.